cypress/integration/inventory/create-edit-delete-material-types.spec.js

```diff
@@ -19,17 +20,17 @@ describe('ui-inventory: Create, edit, delete material types', () => {
   });
 
   after(() => {
-    users.deleteViaApi(userId);
+    Users.deleteViaApi(userId);
   });
 
   it('C505 Settings (Inventory): Create, edit, delete material types', { tags: [TestTypes.smoke] }, () => {
     cy.visit(SettingsMenu.materialTypePath);
     InventorySettings.checkAvailableOptions();
-    MaterialTypesSettings.create(materialTypeName);
-    MaterialTypesSettings.isPresented(materialTypeName);
-    MaterialTypesSettings.edit(newMaterialTypeName);
-    MaterialTypesSettings.delete(newMaterialTypeName);
-    MaterialTypesSettings.isDeleted(newMaterialTypeName);
-    MaterialTypesSettings.verifyMessageOfDeteted(newMaterialTypeName);
+    NewMaterialType.create(materialTypeName);
+    MaterialTypes.isPresented(materialTypeName);
+    MaterialTypes.edit(newMaterialTypeName);
+    MaterialTypes.delete(newMaterialTypeName);
+    MaterialTypes.isDeleted(newMaterialTypeName);
+    MaterialTypes.verifyMessageOfDeteted(newMaterialTypeName);
```

Comment:
It seems like this method needs to be renamed because of 'Deteted'

Reply:
Done